cloud-tasks/function/index.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright 2019 Google LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+/**
+ * Responds to an HTTP request from Cloud Tasks and sends an email using data
+ * from the request body.
+ */
+exports.sendPostcard = async (req, res) => {
+  const sgMail = require('@sendgrid/mail');
```

Comment:
It's usually customary to put requires at the top of the file.

Reply:
I'd still put it up top so that the SG client is only loaded once per file load vs method invocation.